test/test_node_handler.py

```diff
@@ -358,6 +373,7 @@ def test_get_root_node_endpoint(mock_db_find_by_id, mock_init_sub_id):
             _id="61bda8f2eb1a63d2b7152410",
             kind="node",
             name="root",
+            path=["parent", "something", "root"],
```

Comment:
`checkout` node is itself the root node. Hence the `node_obj` and `root_node_obj` should be the same.
Or else we can change the `node_obj` to be different and `root_node_obj` will be the `checkout` node.